app/views/buyers/users/index.html.slim

```diff
@@ -0,0 +1,41 @@
+- content_for :menu do
+  = render 'buyers/accounts/menu'
+
+- content_for :page_header_title, "Users of #{@account.org_name}"
+
+div class="pf-c-card"
+  table class="data pf-c-table" role="grid" id="buyer_users"
+    thead
+      tr role="row"
+        th role="columnheader" scope="col" Name
+        th role="columnheader" scope="col" Email
+        th role="columnheader" scope="col" Created on
+        th role="columnheader" scope="col" Role
+        th role="columnheader" scope="col" colspan="3" State
+        td
```

Comment:
I don't see this `td` does something, might be a typo?

Reply:
![Screenshot 2023-06-16 at 11 20 37](https://github.com/3scale/porta/assets/11672286/22762c1c-8696-4a89-afc9-def50ab0ffa6)